### PR TITLE
No more cross map punches

### LINKED ISSUE #44159: gloves of the north star have no maximum range
## Round ID:

N/A

## Testmerges:

N/A

## Reproduction:

obtain gloves of the north star
set warcry to "*me"
put them on, set harm intent, walk up to someone and click them as many times as you'd like
each hit triggers a *me prompt, don't fill them out or anything
take the gloves off, put them in your bag or the disposals, whatever
walk to any distance and/or location you want (at least on the same z-level)
click cancel on every *me prompt for 1 attack per

### PR DESCRIPTION
**Pull request by Garen7**
## About The Pull Request

Fixes #44159 

## Changelog
:cl: Garen
fix: You can no longer do custom emotes with fists of the north star.
/:cl:

I'm not sure if *me was the only way to do an emote, can someone verify that it is the only way to do an emote with 6 characters? Also before anyone asks this still allows normal emotes like *laugh to be used, it just prevents custom emotes.

### CODE REVIEW

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -75,5 +75,7 @@
 
 /obj/item/clothing/gloves/rapid/attack_self(mob/user)
 	var/input = stripped_input(user,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
+	if(input == "*me") //If they try to do a *me emote it will stop the attack to prompt them for an emote then they can walk away and enter the emote for a punch from far away
+		to_chat(user, "Invalid battlecry, please use another.")
```
Comment (SpaceManiac) [inline code/modules/clothing/gloves/miscellaneous.dm]: you do have to actually cancel here

Comment (Garen7) [reply code/modules/clothing/gloves/miscellaneous.dm]: fuck thanks i forgot

**Comment by ATH1909** [comment]
B-but, my omae wa mou shindeiru memes...

**Comment by Ty-the-Smonk** [comment]
> B-but, my omae wa mou shindeiru memes...

Fist gloves should really just give be a modified plasma fist since it doesn't have stuns, also make it 20 TC for a meme.

**Comment by necromanceranne** [comment]
I was scared this would be a nerf to The Clapper. Glad to see everything is a-okay

**Pull request merged**